libminifi/include/utils/GeneralUtils.h

```diff
@@ -111,10 +106,6 @@ template<typename T>
 using remove_cvref_t = typename std::remove_cv<typename std::remove_reference<T>::type>::type;
 #endif
 
-}  // namespace utils
-}  // namespace minifi
-}  // namespace nifi
-}  // namespace apache
-}  // namespace org
+bool implies(bool a, bool b) noexcept;
```

Comment:
This is trivial enough to keep it inline IMO. And maybe even `constexpr`.
```suggestion
inline constexpr bool implies(bool a, bool b) noexcept { return !a || b; }
```

Reply:
Updated in e2229530a19db98130d0d7007e11e68d62174271